composer.json

```diff
@@ -19,17 +19,17 @@
     ],
     "license": "MIT",
     "require": {
-        "illuminate/support": "^8.0",
-        "illuminate/container": "^8.0",
-        "illuminate/database": "^8.0",
-        "illuminate/events": "^8.0",
-        "mongodb/mongodb": "^1.6"
+        "illuminate/support": "9.x-dev",
+        "illuminate/container": "9.x-dev",
+        "illuminate/database": "9.x-dev",
+        "illuminate/events": "9.x-dev",
+        "mongodb/mongodb": "^1.11"
```

Comment:
Can you take the dev tag for this since laravel 9 is out?

Reply:
Of course but we won't be able to merge it as we don't have enough permissions to do so.

Thanks!